packages/compass/src/app/components/home.tsx

```diff
@@ -332,35 +336,45 @@ function Home({
   );
 
   return (
-    <FileInputBackendProvider createFileInputBackend={createFileInputBackend}>
-      <ConnectionStorageProvider value={connectionStorage}>
-        <ConnectionsManagerProvider value={connectionsManager.current}>
-          <CompassConnections
-            onConnectionAttemptStarted={onConnectionAttemptStarted}
-            onConnectionFailed={onConnectionFailed}
-            onConnected={onConnected}
-            __TEST_INITIAL_CONNECTION_INFO={__TEST_INITIAL_CONNECTION_INFO}
-          >
-            <CompassInstanceStorePlugin>
-              <FieldStorePlugin>
-                {multiConnectionsEnabled && (
-                  <AppRegistryProvider scopeName="Multiple Connections">
-                    <Workspace
-                      // Workspace receives the singleConnectionConnectionInfo
-                      // to wrap the "My Queries" workspace with a
-                      // ConnectionInfoProvider. This makes sure that "My
-                      // Queries" can continue to work when FF for
-                      // multi-connection is not enabled.
-                      singleConnectionConnectionInfo={
-                        connectionInfo ?? undefined
-                      }
-                      onActiveWorkspaceTabChange={onWorkspaceChange}
-                    />
-                  </AppRegistryProvider>
-                )}
-                {!multiConnectionsEnabled &&
-                  (isConnected ? (
-                    <AppRegistryProvider scopeName="Single Connection">
+    <>
+      {isConnected && connectionInfo && (
```

Comment:
Before was always rendered in the menu as main process is not tracking any connection status information